docs/api.md

```diff
@@ -543,6 +544,13 @@ $ node -p "require('pino')().levels"
 
 * See [`logger.level`](#level)
 
+<a id="serializers"></a>
+### logger\[Symbol.for('pino.serializers')\]
+
+Returns the serializers for the logger instance, as seen by the logger
+itself. Child loggers might reuse the same instance of their parent, if
+there are no changes.
```

Comment:
Returns the serializers as applied to the current logger instance. If a child logger did not
register it's own serializer upon instantiation the serializers of the parent will be returned.